test/files/instrumented/indy-symbol-literal.check

```diff
@@ -1,6 +0,0 @@
-indy-symbol-literal.scala:10: warning: a pure expression does nothing in statement position; multiline expressions might require enclosing parentheses
-      'foo.name
-           ^
-indy-symbol-literal.scala:6: warning: a pure expression does nothing in statement position; multiline expressions might require enclosing parentheses
-    'warmup
-    ^
```

Comment:
Was this an accidentally added file in #5310, or was there a test to go along with it that was missed?

Reply:
History of `instrumented` shows the test was deleted by https://github.com/scala/scala/pull/6864

Reply:
Ok, I'll keep it deleted, then.